appinfo/info.xml

```diff
@@ -35,7 +35,7 @@ Before you update to a new version, [check the changelog](https://github.com/nex
         <lib>SimpleXML</lib>
         <lib>iconv</lib>
         <owncloud max-version="0" min-version="0"/>
-        <nextcloud min-version="13" max-version="14"/>
+        <nextcloud min-version="14" max-version="14"/>
```

Comment:
Does this no longer work with the 13 version?

Reply:
No. Behaviour is not retro compatible

Reply:
Since this will always make travis fail, maybe the branch bump for travis should also be part of this review (which means the final merger would have to wait for the stable14 branch to exist). @nextcloud/news anyone else with an opinion about this?

Reply:
For now you can just test against master until there is a stable14 branch.

Reply:
Wouldn't it make sense to have a stable13 branch for news? And from master then the 14 app release can be done?
So bugfixes for 13.x app can still be merged & released (if needed)

Reply:
@tobiasKaminsky I guess?

Reply:
In the past, Bernhard always dropped support for older server releases once a new major release was out. Doesn't mean it stil needs to be done like that, but it'd be easier and follows the known pattern.